client/app/components/dynamic-form/DynamicForm.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Input, InputNumber, Checkbox, Button, Upload, Icon } from 'antd';
```

Comment:
Unfortunately [this](https://github.com/getredash/redash/pull/3247#discussion_r245055244) 😓

Reply:
I saw it a few days ago :confused:, anyway I'll separate those...